### PR TITLE
Removes references for headless dc

### PR DESCRIPTION
**Pull request by aliciakrebs**
### Description
Remove references to headless DC and deprecate content. 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/headless.md
Comment (lysannep) [inline docs/headless.md:1]: Since we've added a redirect, I think this page can be removed.

**Review by lysannep — APPROVED** [review]
Seal-of-Approval

File: docs/web-apps/automated-testing/cypress/yaml/v1.md [hunk 1]
```diff
@@ -911,7 +897,7 @@ To split tests in the most efficient way possible, use:
 
 - `spec` when the number of specs is less than the configured concurrency.
 - `concurrency` when the number of specs is larger than the configured concurrency.
-:::
+  :::
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/cypress/yaml/v1.md:900]:
```suggestion
:::
```

File: docs/web-apps/automated-testing/cypress/yaml/v1.md [hunk 2]
```diff
@@ -934,7 +920,7 @@ Filtering relies on the env variables that are used by `@cypress/grep`.
 
 - `grep` to filter on test name.
 - `grepTags` to filter on test tags.
-:::
+  :::
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/cypress/yaml/v1.md:923]:
```suggestion
:::
```

File: docs/web-apps/automated-testing/cypress/yaml/v1alpha.md [hunk 3]
```diff
@@ -916,7 +902,7 @@ To split tests in the most efficient way possible, use:
 
 - `spec` when the number of specs is less than the configured concurrency.
 - `concurrency` when the number of specs is larger than the configured concurrency.
-:::
+  :::
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/cypress/yaml/v1alpha.md:905]:
```suggestion
:::
```

File: docs/web-apps/automated-testing/testcafe/yaml.md [hunk 4]
```diff
@@ -759,7 +747,7 @@ To split tests in the most efficient way possible, use:
 
 - `spec` when the number of specs is less than the configured concurrency.
 - `concurrency` when the number of specs is larger than the configured concurrency.
-:::
+  :::
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/testcafe/yaml.md:750]:
```suggestion
:::
```

File: docs/web-apps/automated-testing/cypress/yaml/v1.md [hunk 5]
```diff
@@ -883,20 +883,6 @@ excludeSpecPattern: ['**/*.*']
 
 ---
 
-### `headless`
```
Comment (mhan83) [inline docs/web-apps/automated-testing/cypress/yaml/v1.md:886]: References to `headless` in saucectl configs is framework specific config and unrelated to sauce's `headless` product. So this should not be removed. Same applies to the changes to the v1alpha doc, playwright and testcafe docs.

**Review by mhan83 — CHANGES_REQUESTED** [review]

File: docs/web-apps/automated-testing/cypress/yaml/v1.md (hunk 5 above)
Comment (aliciakrebs) [reply docs/web-apps/automated-testing/cypress/yaml/v1.md:886]: @mhan83 thanks. I've updated to only remove the links to the page for the Sauce Headless product.

**Review by alexplischke — APPROVED** [review]

**Review by mhan83 — APPROVED** [review]

**Pull request merged**